### ISSUE TITLE
Institution filter to exclude non-UK institutions

### ISSUE THREAD

**Issue by TaraWainwright** [gbif/hp-uk-collections#3]
Currently, when filtering by institution on the specimen search page (https://uk-collections.hp.gbif-staging.org/specimen/search), non-UK institutions are suggested. 

It would be helpful if we could limit the suggested institutions to only UK institutions, as selecting a non-UK institution won't produce any specimens and may be confusing.

**Comment by MortenHofft** [comment]
Makes sense, I've created an issue to add country filtering to the suggest endpoint https://github.com/gbif/registry/issues/451
